src/fsharp/service/SemanticClassification.fs

```diff
@@ -116,61 +129,198 @@ module TcResolutionsExtensions =
                     // 'seq' in 'seq { ... }' gets colored as keywords
                     | (Item.Value vref), ItemOccurence.Use, _, _, _, m when valRefEq g g.seq_vref vref ->
                         add m SemanticClassificationType.ComputationExpression
+
                     | (Item.Value vref), _, _, _, _, m when isValRefMutable vref ->
                         add m SemanticClassificationType.MutableVar
+
                     | Item.Value KeywordIntrinsicValue, ItemOccurence.Use, _, _, _, m ->
                         add m SemanticClassificationType.IntrinsicFunction
+
                     | (Item.Value vref), _, _, _, _, m when isFunction g vref.Type ->
                         if valRefEq g g.range_op_vref vref || valRefEq g g.range_step_op_vref vref then 
                             ()
                         elif vref.IsPropertyGetterMethod || vref.IsPropertySetterMethod then
                             add m SemanticClassificationType.Property
+                        elif vref.IsMember then
+                            add m SemanticClassificationType.Method
                         elif IsOperatorName vref.DisplayName then
                             add m SemanticClassificationType.Operator
                         else
                             add m SemanticClassificationType.Function
-                    | Item.RecdField rfinfo, _, _, _, _, m when isRecdFieldMutable rfinfo ->
-                        add m SemanticClassificationType.MutableVar
-                    | Item.RecdField rfinfo, _, _, _, _, m when isFunction g rfinfo.FieldType ->
-                        add m SemanticClassificationType.Function
-                    | Item.RecdField EnumCaseFieldInfo, _, _, _, _, m ->
-                        add m SemanticClassificationType.Enumeration
-                    | Item.MethodGroup _, _, _, _, _, m ->
-                        add m SemanticClassificationType.Function
-                    // custom builders, custom operations get colored as keywords
+
+                    | (Item.Value vref), _, _, _, _, m ->
```

Comment:
Need to account for discards here, they are currently colored as values but they should probably just be plaintext